samples/apps/copilot-chat-app/README.md

```diff
@@ -0,0 +1,77 @@
+3.	The sample application requires .NET 6.0 or greater.  
+4.	Node.js should be installed, 
+5.	Yarn will be required.
+
+
+## Running the Sample
+1.	You will need an Open AI Key or Azure Open AI Service Key for this sample.
+2.	You will need to register your application in the Azure Portal.  Follow the steps to register your app here.
+
+    1. Select Single-page application (SPA) as platform type, and the redirect URI will be `http://localhost:3000`
+    2.	Select `Accounts in any organizational directory and personal Microsoft Accounts` as supported account types for this sample.
+    3. Make a note of this Application (client) ID from the Azure Portal, we will make use of it later.
+3.	The sample uses two applications, a front-end web UI, and a back-end server.  First, let’s set up and verify the back-end server is running.
+
+    1. Navigate to `\samples\apps\copilot-chat-app\SKWebApi`
+    2.	Update `appsettings.json` with these settings:
+
+          *	Under the `“ServicePort”` block, make the following configuration changes to match your instance:
+
+            * `“AIService”: “AzureOpenAI”`, or whichever option is appropriate for your instance.
+            * `“DeploymentOrModelID”: “text-davinci-003”,` or whichever option is appropriate for your instance.  
+            * `“Endpoint”:` “Your Endpoint address, i.e. http://project.region.azure.com”
+            * `“Key”:` “Your azure endpoint key, i.e. a 36+ digit hex code” 
+
+        * Under the `“EmbeddingConfig”` block, make sure the following configuration changes to match your instance:
+            * `“AIService”: “AzureOpenAI”,` or whichever option is appropriate for your instance.
+            * `“DeploymentOrModelID”: “text-embedding-ada-002”,` or whichever option is appropriate for your instance.    
+            *	`“Key”:` “Your azure endpoint key, i.e. a 36+ digit hex code” 
+            
+4. Build the back-end server by following these instructions:
+    1.	In the terminal navigate to  `\samples\apps\copilot-chat-app\SKWebApi`
+    2.	Execute the command `dotnet build`
+    3.	Once the build is complete, Execute the command `dotnet run`
+    4.	Test the back-end server to confirm it is running.
+        * Open a web browser, and navigate to `https://localhost:40443/probe`
+        * You should see a confirmation message: `Semantic Kernel service is up and running`
+>Note: you may need to accept the locally signed certificate on your machine in order to see this message.  It is important to do this, as your browser may need to accept the certificate before allowing the WebApp to communicate with the backend.
+
+>Note: You may need to acknowledge the Windows Defender Firewall, and allow the app to communicate over private or public netowrks as appropriate.
+
+
+5.	Now that the back-end server is setup, and confirmed operating, let’s proceed with setting up the front-end WebApp.
+    1. Navigate to `\apps\copilot-chat-app\webapp`
```

Comment:
There are spacing inconsistencies in this section.

Reply:
Resolved.